jupyterhub_traefik_proxy/proxy.py

```diff
@@ -293,19 +293,30 @@ def _warn_empty_username(self):
         )
         return ""
 
-    traefik_api_hashed_password = Unicode()
+    traefik_api_hashed_password = Unicode(
+        config=True,
+        help="""
+        Set the hashed password to use for the API
+
+        If unspecified, `traefik_api_password` will be hashed with bcrypt.
+        ref: https://doc.traefik.io/traefik/middlewares/http/basicauth/
+        """,
+    )
+
+    @default("traefik_api_hashed_password")
+    def _generate_htpassword(self):
+        import bcrypt
```

Comment:
Can this be a top-level import? Or is the loading time prohibitive?

Reply:
I think it was here because it was maybe an optional dependency when this auth stuff went in the static config pre-traefik 3, so it was avoiding an import of a package that may not be used. Now that it's required, moving it top-level should be fine.